runtime/src/bucket_map_holder.rs

```diff
@@ -3,18 +3,24 @@ use crate::bucket_map_holder_stats::BucketMapHolderStats;
 use crate::in_mem_accounts_index::SlotT;
 use crate::waitable_condvar::WaitableCondvar;
 use solana_bucket_map::bucket_map::{BucketMap, BucketMapConfig};
+use solana_sdk::timing::AtomicInterval;
 use std::fmt::Debug;
 use std::sync::atomic::{AtomicBool, AtomicU8, AtomicUsize, Ordering};
 use std::sync::Mutex;
 pub type Age = u8;
 
+const AGE_MS: u64 = 400; // 400 ms per age increment, which matches slot time
```

Comment:
Is there already a slot time constant somewhere? Or, is the intent to have the Age time be tunable and different from the slot time?

Reply:
I wanted it different, but logically the same. Maybe it does make sense to use the clock constant. I'll investigate.

Reply:
I made it the same constant